### PR TITLE
Include concluded information for courses

### PR DESCRIPTION
**Pull request by bgarciaentornos**

### CODE REVIEW

File: src/main/java/edu/ksu/canvas/model/Course.java [hunk 1]
```diff
@@ -466,6 +467,11 @@ public void setBlueprint(Boolean blueprint) {
         this.blueprint = blueprint;
     }
 
+    @CanvasField(postKey = "concluded")
```
Comment (mpellicer) [inline src/main/java/edu/ksu/canvas/model/Course.java:470]: Do we need the postKey?

**Comment by bgarciaentornos** [comment]
Removed postKey

**Review by mpellicer — APPROVED** [review]
This looks good to me, thanks.

**Comment by mpellicer** [comment]
A unit test would be nice but I don't see unit tests for other fields, the sample JSON looks very incomplete to me.

**Comment by bgarciaentornos** [comment]
I thought about it but there's already a unit test for the include course_image field, all the other include cases work the same way and don't have a test.

**Pull request merged**